org.gnome.pushupmaster.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id" : "org.gnome.pushupmaster",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
```

Comment:
this version is not available on flathub

Reply:
ah. ok. I need to check then, if it works with an older version, or I have to wait,